webapp/sources/rudder/rudder-core/src/main/scala/com/normation/rudder/services/policies/DataStructures.scala

```diff
@@ -357,7 +357,7 @@ final case class Policy(
   // == .toList (keep order)     ==> List[List[Variable]]
   // == flatten (keep order)     ==> List[Variable]
   val expandedVars    = Policy.mergeVars(policyVars.map( _.expandedVars.values).toList.flatten)
```

Comment:
i'm really tempted to make a def out of this one, as it is used only in two different place